README.md

```diff
@@ -56,3 +57,6 @@ you are good to go.
 
 ## Dependencies
 `arXbib` is written in Python3.4 and needs the [beautifulsoup](http://www.crummy.com/software/BeautifulSoup/bs4/doc/) parser.
+
+[inspire]:    https://inspirehep.net/
```

Comment:
don't like to have link-duplication. and conda was just on the same visible page … but really no strong opinion do as you wish